### ISSUE TITLE
Angle properties aren't displayed in degrees in `DISABLE_DEPRECATED` builds

### ISSUE THREAD

**Issue by huwpascoe** [godot-jolt/godot-jolt#961]
It's clear that the intention was to be degrees as the range is -180+180
```cpp
BIND_PROPERTY_RANGED("limit_upper", Variant::FLOAT, "-180,180,0.1,radians");
BIND_PROPERTY_RANGED("limit_lower", Variant::FLOAT, "-180,180,0.1,radians");
...
BIND_PROPERTY_RANGED("swing_limit_span", Variant::FLOAT, "-180,180,0.1,radians");
BIND_PROPERTY_RANGED("twist_limit_span", Variant::FLOAT, "-180,180,0.1,radians");
...
// and so on
```

**Comment by mihe** [comment]
Where exactly are you seeing radians instead of degrees? The properties you mentioned are displayed in degrees in the property inspector, but stored as radians (and set as radians through code).

The `radians` hint in Godot is (or rather was when I wrote that code) somewhat ambiguously named, as it tells Godot to store that particular property in radians but to display it in degrees, and the limits specified in the property hint are supposed to be in the display format.

Nowadays the `radians` property hint is instead called `radians_as_degrees`, but `radians` remains an alias for it.

**Comment by huwpascoe** [comment]
I noticed it with the hinge joint, spent a while trying to figure out why it wasn't limiting then discovered it was displaying as radians.

Forgot to mention: running on Godot 4.3

**Comment by huwpascoe** [comment]
Okay I found out what's happened. It's actually deprecated, I use a build with DISABLE_DEPRECATED flag so it no longer works in that case.

**Comment by mihe** [comment]
I just took a look and the `JoltHingeJoint3D` node displays its limits in degrees but stores them as radians, same as the default `HingeJoint3D`, as well as the other Jolt substitutes.

Is there some other step involved?

**Comment by mihe** [comment]
> Okay I found out what's happened. It's actually deprecated, I use a build with DISABLE_DEPRECATED flag so it no longer works in that case.

Ah, that's interesting. I didn't realize `radians` was deprecated.

I'll go ahead and switch over to `radians_as_degrees` then.

**Comment by mihe** [comment]
The fix for this can now be found in the newly released [0.14.0-stable](https://github.com/godot-jolt/godot-jolt/releases/tag/v0.14.0-stable) release.

Note that the release on Godot Asset Library is still pending approval, so you might need to download it from GitHub for the time being.
